tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
@yannickcr No, I'm saying add `getParser()` on ESLintTester.

@ilyavolodin I'm not comfortable with that type of testing - you're relying on implicit failures to determine something that should be explicit. We should verify that the parsing succeeds, and also verify that the name of the parser was used as expected.